test/rules/no-exclusive-tests.js

```diff
@@ -26,43 +26,53 @@ ruleTester.run('no-exclusive-tests', rules['no-exclusive-tests'], {
     invalid: [
         {
             code: 'describe.only()',
-            errors: [ { message: expectedErrorMessage, column: 10, line: 1 } ]
+            errors: [ { message: expectedErrorMessage, column: 10, line: 1 } ],
+            output: 'describe()'
         },
         {
             code: 'describe["only"]()',
-            errors: [ { message: expectedErrorMessage, column: 10, line: 1 } ]
+            errors: [ { message: expectedErrorMessage, column: 10, line: 1 } ],
+            output: 'describe()'
         },
         {
             code: 'it.only()',
-            errors: [ { message: expectedErrorMessage, column: 4, line: 1 } ]
+            errors: [ { message: expectedErrorMessage, column: 4, line: 1 } ],
+            output: 'it()'
         },
         {
             code: 'it["only"]()',
-            errors: [ { message: expectedErrorMessage, column: 4, line: 1 } ]
+            errors: [ { message: expectedErrorMessage, column: 4, line: 1 } ],
+            output: 'it()'
         },
         {
             code: 'suite.only()',
-            errors: [ { message: expectedErrorMessage, column: 7, line: 1 } ]
+            errors: [ { message: expectedErrorMessage, column: 7, line: 1 } ],
+            output: 'suite()'
         },
         {
             code: 'suite["only"]()',
-            errors: [ { message: expectedErrorMessage, column: 7, line: 1 } ]
+            errors: [ { message: expectedErrorMessage, column: 7, line: 1 } ],
+            output: 'suite()'
         },
         {
             code: 'test.only()',
-            errors: [ { message: expectedErrorMessage, column: 6, line: 1 } ]
+            errors: [ { message: expectedErrorMessage, column: 6, line: 1 } ],
```

Comment:
Could explain to me the counting mechanism for the `column`? For `test.only` and `test["only"]` the number is in both cases `6` here, while these definitely don't have the same starting column of the `o`.